himin/day2/두 정수 사이의 합.js

```diff
@@ -0,0 +1,11 @@
+function solution(a, b) {
+    let countAB = 0;
+    
+    const min = Math.min(a, b);
+    const max = Math.max(a, b);
```

Comment:
값이 변하지 않는 상수가 항상 코드의 상단으로 올라오는 것이 암묵적인 룰입니다 :) 
```suggestion
    const min = Math.min(a, b);
    const max = Math.max(a, b);
    let countAB = 0;
```

Reply:
https://github.com/modern-agile-team/9term-assignment/commit/498f1d69f7451b6242ee8bdd456669fbd67ced78

@ssi02014 세심한 피드백 감사드립니다!